packages/transaction-controller/jest.config.js

```diff
@@ -18,9 +18,9 @@ module.exports = merge(baseConfig, {
   coverageThreshold: {
     global: {
       branches: 84.22,
-      functions: 92.95,
-      lines: 95.74,
-      statements: 95.82,
+      functions: 92.9,
+      lines: 95.54,
+      statements: 95.63,
```

Comment:
Feels like we should aim to keep these about the same within this PR if possible?

Reply:
Though I understand we don't wanna make this PR bigger. I'll try to see where coverage has dropped